examples/service-with-lifecycle.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myservice-pod
+  annotations:
+    podDirector: "director1"
+    podEnvironment: "dev"
+    podDC: "-dc1"
+    podWeight: "1"
+    vaasUser: "admin"
```

Comment:
It would be better to move vaasUser, vaasKey and vaasUrl to k8s config maps or secrets? It would be easier to update it in the future.